llama-index-packs/llama-index-packs-rag-evaluator/llama_index/packs/rag_evaluator/base.py

```diff
@@ -44,6 +46,7 @@ def __init__(
         judge_llm: Optional[LLM] = None,
         embed_model: Optional[BaseEmbedding] = None,
         show_progress: bool = True,
+        result_path = None,
```

Comment:
missing type: Should be `Optional[str]`?

Reply:
Good point, thx for fixing @logan-markewich